pkgs/development/compilers/llvm/3.9/clang/default.nix

```diff
@@ -17,7 +17,10 @@ let
 
     cmakeFlags = [
       "-DCMAKE_CXX_FLAGS=-std=c++11"
+      "-DWITH_POLLY:Bool=true"
+      "-DLINK_POLLY_INTO_TOOLS:Bool=true"            "-DLLVM_LINK_LLVM_DYLIB:Bool=true"
     ] ++
+#    (stdenv.lib.optional llvm.enableSharedLibraries "-DLLVM_LINK_LLVM_DYLIB:Bool=true") ++
```

Comment:
how can i do this? That is, how to reference the input argument to a build dependency?